src/website/app/ErrorBoundary.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React, { Component } from 'react';
+import { createStyledComponent } from '../../styles';
+import { createThemedComponent } from '../../themes';
+import _Button from '../../Button';
+import Heading from './Heading';
+
+type Props = {
+  buttonLabel: string,
+  children?: React$Node,
+  errorMessage: string,
+  onReRender?: () => void
+};
+
+type State = {
+  hasError: boolean
+};
+
+const ErrorBlock = createStyledComponent('div', ({ theme }) => ({
+  backgroundColor: '#fce3e3', // color.red_10
+  border: '1px solid #fabebe', // color.red_20
+  color: theme.color_text_danger,
+  lineHeight: theme.lineHeight_prose,
+  padding: theme.space_inset_lg,
+  textAlign: 'center'
+}));
+
+const Button = createThemedComponent(_Button, {
+  borderColor_focus: '#db2929' // color.red_60
+});
+
+export default class ErrorBoundary extends Component<Props, State> {
+  static defaultProps = {
+    errorMessage: 'Oops, something went wrong.',
```

Comment:
I think it would be helpful if this pointed the user to open their dev console to see the error. The errors in the console are super helpful for fixing the problem so it would be unfortunate if someone didn't know to check there.

Reply:
Great idea! Done.